subsys/bluetooth/controller/util/util.c

```diff
@@ -215,3 +216,58 @@ int util_aa_le32(uint8_t *dst)
 
 	return 0;
 }
+
+#if defined(CONFIG_BT_CTLR_ADV_ISO)
+void util_saa_le32(uint8_t *dst, uint8_t handle)
+{
+	uint32_t saa, saa_15, saa_16;
+	uint16_t max;
+	uint8_t bits;
+
+	lll_csrand_get(dst, sizeof(uint32_t));
+	saa = sys_get_le32(dst);
+
+	saa_15 = (saa >> 15) & 0x01;
+	saa &= ~BIT(19);
+	saa |= saa_15 << 19;
+	saa &= ~BIT(16);
+	saa_16 = ~saa_15 & 0x01;
+	saa |= saa_16 << 16;
+	saa &= ~BIT(22);
+	saa |= saa_16 << 22;
+	saa &= ~BIT(25);
+	saa |= BIT(23);
+
+	/* For any pair of BIGs transmitted by the same device, the SAA 15-0
+	 * values shall differ in at least two bits.
+	 */
```

Comment:
Would add a comment describing how this is achieved, i.e.
1. Find how many bits are...
2. Clear those bits...
3. Add a unique value based on...

Reply:
done